NEWS.md

```diff
@@ -11,6 +11,7 @@
 * Rename `semicolon_terminator_linter` to `semicolon_linter` for better consistency. `semicolon_terminator_linter` survives but is marked for deprecation. The new linter also has a new signature, taking arguments `allow_compound` and `allow_trailing` to replace the old single argument `semicolon=`, again for signature consistency with other linters.
 * Combined several curly brace related linters into a new `brace_linter` (#1041, @AshesITR):
   + `closed_curly_linter()`
+  + `open_curly_linter()`, no longer linting unnecessary trailing whitespace
```

Comment:
let's add a test around this? it looks like `lintr::lint(text = "function(x) {   \n  x\n}", linters = list(open_curly_linter(), trailing_whitespace_linter()))` returns two lints before this fix. Let's test to ensure there's only one.

Reply:
The problem still persists (because I left `open_curly_linter()` as-is, but we can test that `brace_linter()` doesn't lint said expression.

Reply:
yep, a test with brace_linter is what I had in mind